src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/EventModel.java

```diff
@@ -43,12 +40,12 @@ public void setSummary(String summary) {
     this.summary = summary;
   }
 
-  public String getId() {
-    return id;
+  public String getEvent_id() {
+    return event_id;
   }
 
-  public void setId(String id) {
-    this.id = id;
+  public void setEvent_id(String event_id) {
+    this.event_id = event_id;
   }
```

Comment:
Please follow the styleguide, variables should be always lowerCamelCase. Please correct.